controllers/misc.controllers.js

```diff
@@ -0,0 +1,7 @@
+const fs = require("fs/promises");
+
+exports.getEndpoints = (req, res, next) => {
+  return fs.readFile("./endpoints.json", "utf8").then((result) => {
```

Comment:
```js
  return fs.readFile("./endpoints.json", "utf8").then((result) => {
```
Be careful of the `.` in your file path here. Could cause problems if the terminal is not in the root dir. Consider using `__dirname` instead